pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
+wi.SetClient(
+    &http.Client{
+        Timeout: time.Second * 2,
+    },
+    nil,
+)
+```
+
+If your service will use any internal function of william like a service, Ej: adding permission directly from your service and not from the Hub you need to set a KeyPair to auth.
+
+```go
+wi.SetKey(
+    "ServiceKeyID",
+    "ServiceKeySecret",
+)
+```
+
+By default William is created with a bypass, you can enable with:
+```go
+wi.Enable()
+```
+
+## Permissions Handler
+You want to validate if a user have a valid permission for a specific Handler. If your application don't care about resource you can use william.Generate("RL", "Action") that will wrap this for you.
```

Comment:
I'm not sure I understood what this part. What do you mean by 

> If your application don't care about resource you can use william.Generate("RL", "Action") that will wrap this for you.

Reply:
On william you have 4 parts, Service, Owner/Lender, Action and Resources. If you don’t care about the resource, Generate return permission with an * in the resources field.

Reply:
This paragraph is a bit wonky - the first sentence doesn't connect to the rest and we could explain better what we mean by "resource" (it's a highly overloaded term).